src/cli/args.rs

```diff
@@ -61,8 +61,18 @@ pub enum Subcommand {
         output: PathBuf,
 
         /// Compression level, applied to all formats
-        #[arg(short, long)]
+        #[arg(short, long, group = "compression-level")]
         level: Option<i16>,
+
+        /// Fastest compression level possible,
+        /// conflicts with --level and --slow
```

Comment:
I think it's not necessary to add the conflict message to --help, what do you think?

Reply:
it is in `--help`

```
      --fast             Fastest compression level possible, conflicts with --level and --slow
      --slow             Slowest (and best) compression level possible, conflicts with --level and --fast
```